### ISSUE TITLE
Add a startup page (loading indicator, e. g. bigger models)

### ISSUE THREAD

**Issue by MoritzWeber0** [DSD-DBS/capella-model-explorer#45]
When using the new feature to compare models with large models, it can take some time until the application starts successfully. 

Until then, the application doesn't respond to requests. This might be confusing for users since they don't know what the state of the application is. During loading of the model, a startup screen should be displayed. Once the model is loaded and the backend is ready, the user should be redirected to the usual landing page.

**Comment by Wuestengecko** [comment]
No longer relevant.
